packages/common/src/grpc-service.js

```diff
@@ -158,6 +159,18 @@ function GrpcService(config, options) {
     this.grpcCredentials = grpc.credentials.createInsecure();
   }
 
+  this.grpcMetadata = null;
```

Comment:
Have you noticed any system test failures in the event that grpcMetadata is `null`? I pulled your branch down and linked common to all other packages and was met with an immediate Bigtable failure.

Doing this fixed the issue for me locally.

```js
this.grpcMetadata = new grpc.Metadata();

if (config.grpcMetadata) {
  for (var prop in config.grpcMetadata) {
    if (config.grpcMetadata.hasOwnProperty(prop)) {
      this.grpcMetadata.add(prop, config.grpcMetadata[prop]);
    }
  }
}
```